web/pgadmin/browser/server_groups/servers/databases/schemas/collations/static/js/collation.ui.js

```diff
@@ -87,7 +87,7 @@ export default class CollationSchema extends BaseUISchema {
       },
       {
         id: 'provider', label: gettext('Locale Provider'),
-        editable: false, type: 'select',mode: ['create', 'edit'], group: gettext('Definition'),
+        editable: false, type: 'select',mode: ['create'],
```

Comment:
Any reason you removed, group? On General tab Locale provider field is showing up.